app/repositories/onlinetesting/OnlineTestRepository.scala

```diff
@@ -171,15 +171,24 @@ trait OnlineTestRepository extends RandomSelection with ReactiveRepositoryHelper
     selectOneRandom[NotificationExpiringOnlineTest](query)
   }
 
-  def updateProgressStatus(appId: String, progressStatus: ProgressStatus): Future[Unit] = {
+  def updateProgressStatus(appId: String, progressStatus: ProgressStatus, forceApplicationStatusUpdate: Boolean = true): Future[Unit] = {
```

Comment:
force would suggest that even when false the update is done whereas that is not the case. What do you reckon?